lib/fog/vsphere.rb

```diff
@@ -25,4 +23,8 @@ def self.class_from_string(name, default_path = '')
       nil
     end
   end
+
+  module Compute
+    Vsphere = ::Fog::Vsphere::Compute
```

Comment:
not sure about this nasty alias.

Reply:
Neither am I. Do others have this?
With this PR merged I‘d like to bump the major version. So I guess we don‘t need to stay backwards compatible.

Reply:
Let's fix this once and for all. We'll bump the major version that includes this commit. For Foreman this should not be a problem at all (we might just need to restrict the gem version via a commit in the stable branches).

@tbrisker: Opinions?

Reply:
I was just worried about the even older versions, but I guess it is 1) not our problem 2) not an issue, because we will require higher version of fog in new version, right?

tbrisker has a party today :balloon: so won't answer till tomorrow probably.

Reply:
i'm guessing this is no longer relevant, but whatever you folks think is best for the gem is fine by me :)